dev/aura.html

```diff
@@ -300,6 +338,8 @@
   <body>
     <vaadin-app-layout>
       <header slot="drawer" class="aura-view-header">
+        <vaadin-avatar theme="vaadin square" has-color-index
```

Comment:
While this is just for demo purposes, maybe we need actual API for this so that we don't have to set attribute manually? Using it this way feels wrong as it's an internal state attribute.